openfisca_aotearoa/parameters/disability_allowance/income_limits.yaml

```diff
@@ -0,0 +1,24 @@
+description: Income threshold for getting access to disability allowance
+reference: https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784880.html?search=sw_096be8ed81c90476_disability+allowance_25_se&p=4
```

Comment:
```suggestion
reference: https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784880.html
```

Reply:
thanks @Br3nda ! realised it was an incorrect link so have fixed!